test/test_linalg.py

```diff
@@ -2916,7 +2916,8 @@ def func(root, b, upper):
 
             gradcheck(func, [root, b, upper])
             # TODO(#50743): the following fails with batched grad testing
-            gradgradcheck(func, [root, b, upper], atol=1e-3, check_batched_grad=False)
+            # TODO(#56235): disabling temporarily
+            # gradgradcheck(func, [root, b, upper], atol=1e-3, check_batched_grad=False)
```

Comment:
Does raising atol to 1e-2 work? That might be better than completely removing it?

Reply:
Not quite, for certain seeds the difference can be quite large. I have more details in the linked issue, but basically for 90% of the seeds I sampled, the difference is small i.e., it passes with atol=1e-12, rtol=1e-3. However, there are certain seeds where the randomized inputs produce differences very large as well. So no matter what atol, we set, there would always be a non-negligible chance that it fails.